backend/app/bug_handler.go

```diff
@@ -0,0 +1,133 @@
+package app
+
+import (
+	"errors"
+
+	"github.com/codescalersinternships/Flyspray/models"
+	"github.com/gin-gonic/gin"
+	"github.com/go-playground/validator/v10"
+	"gorm.io/gorm"
+)
+
+func (app *App) createBug(ctx *gin.Context) (interface{}, Response) {
+	var bug models.Bug
+	if err := ctx.BindJSON(&bug); err != nil {
+		return nil, BadRequest(errors.New("failed to read data"))
```

Comment:
- log errors. zerolog is recommended
- return a friendly user errors
apply for all